cluster/images/etcd-version-monitor/etcd-version-monitor.go

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
I do not see it downloading etcdctl? I'd greatly prefer if this did not download or use etcdctl, so that we don't have to worry about the version being out of sync.